lib/ci-helper.ts

```diff
@@ -662,7 +662,8 @@ export class CIHelper {
                     }
                 } catch (reason) {
                     throw new Error(`User ${
-                        accountName} is not a valid GitHub username.`);
+                        accountName} is not a valid GitHub username: ${
+                        reason}`);
```

Comment:
Could you rebase? This branch still has commits that I had squashed.

Reply:
Done